W504 false positive for type annotation `...`

```python
def f(
    match: str = ...
):
    ...
```

```console
$ pycodestyle t.py --select=W504
t.py:2:18: W504 line break after binary operator
$ pycodestyle --version
2.5.0
```

discovered in https://github.com/pytest-dev/pytest/pull/5593/files

--- a/pycodestyle.py
+++ b/pycodestyle.py
@@ -1205,13 +1205,16 @@ def explicit_line_join(logical_line, tokens):
                 parens -= 1
 
 
+_SYMBOLIC_OPS = frozenset("()[]{},:.;@=%~") | frozenset(("...",))
+
+
 def _is_binary_operator(token_type, text):
     is_op_token = token_type == tokenize.OP
     is_conjunction = text in ['and', 'or']
     # NOTE(sigmavirus24): Previously the not_a_symbol check was executed
     # conditionally. Since it is now *always* executed, text may be
     # None. In that case we get a TypeError for `text not in str`.
-    not_a_symbol = text and text not in "()[]{},:.;@=%~"
+    not_a_symbol = text and text not in _SYMBOLIC_OPS
     # The % character is strictly speaking a binary operator, but the
     # common usage seems to be to put it next to the format parameters,
     # after a line break.

--- a/testsuite/python3.py
+++ b/testsuite/python3.py
@@ -37,3 +37,10 @@ class Class:
 
     def m(self):
         xs: List[int] = []
+
+
+# Used to trigger W504
+def f(
+    x: str = ...
+):
+    ...

